it-tests/MavenDependencyUpdate.test.ts

```diff
@@ -158,6 +158,11 @@ describe('Maven dependency update pom.xml on save test', function () {
 		const sqlComponent = await driver.findElement(By.css('g[data-nodelabel="sql"]'));
 		await driver.actions().contextClick(sqlComponent).perform();
 
+		// workaround to force a redraw and have contextual menu correctly visible
+		await kaotoWebview.switchBack();
```

Comment:
should be `workaroundToRedrawContextualMenu(kaotoWebview: WebView)` method from utils.ts called instead of this block?